controllers/recruitersController.js

```diff
@@ -0,0 +1,22 @@
+const recruiterQuery = require('../models/recruiters')
+
+/**
+ * Posts the data about the recruiter
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+
+const addRecruiter = async (req, res) => {
+  try {
+    const result = await recruiterQuery.addRecruiterInfo(req.body, req.params.username)
+    return res.json(result)
+  } catch (error) {
+    logger.error(`Error while adding recruiterInfo: ${error}`)
+    return res.boom.serverUnavailable('Something went wrong please contact admin')
```

Comment:
We should be handling the case when the given username does not exist.
Return 404 in that case using `res.boom.notFound...`

Check this out for more info: https://github.com/Real-Dev-Squad/website-backend/blob/7bc36f4efe28d434f9f55bcbdab10c66bb1f7aed/controllers/auction.js#L73

The model function will return a variable if the user is not found and we can set error code based on it in controller

Reply:
Handled the error for user.